src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -1134,23 +965,235 @@ internal static class Defaults
             int numSummaryTermPerTopic = Defaults.NumSummaryTermPerTopic,
             int numBurninIterations = Defaults.NumBurninIterations,
             bool resetRandomGenerator = Defaults.ResetRandomGenerator)
-            : this(env, new[] { new LatentDirichletAllocationTransformer.ColumnInfo(outputColumnName, inputColumnName ?? outputColumnName,
+            : this(env, new[] { new ColumnInfo(outputColumnName, inputColumnName ?? outputColumnName,
                 numTopic, alphaSum, beta, mhstep, numIterations, likelihoodInterval, numThreads, numMaxDocToken,
                 numSummaryTermPerTopic, numBurninIterations, resetRandomGenerator) })
         { }
 
         /// <include file='doc.xml' path='doc/members/member[@name="LightLDA"]/*' />
         /// <param name="env">The environment.</param>
         /// <param name="columns">Describes the parameters of the LDA process for each column pair.</param>
-        public LatentDirichletAllocationEstimator(IHostEnvironment env, params LatentDirichletAllocationTransformer.ColumnInfo[] columns)
+        internal LatentDirichletAllocationEstimator(IHostEnvironment env, params ColumnInfo[] columns)
         {
             Contracts.CheckValue(env, nameof(env));
             _host = env.Register(nameof(LatentDirichletAllocationEstimator));
             _columns = columns.ToImmutableArray();
         }
 
         /// <summary>
-        /// Returns the schema that would be produced by the transformation.
+        /// Describes how the transformer handles one column pair.
+        /// </summary>
+        public sealed class ColumnInfo
+        {
+            /// <summary>
+            /// Name of the column resulting from the transformation of <cref see="InputColumnName"/>.
+            /// </summary>
+            public readonly string Name;
+            /// <summary>
+            /// Name of column to transform. If set to <see langword="null"/>, the value of the <cref see="Name"/> will be used as source.
```

Comment:
> /// Name of column to transform. If set to <see langword="null"/>, the value of the <cref see="Name"/> will be used as source. [](start = 11, length = 127)

as Zeeshan A pointed in other PRs, you need to left only first sentence. It's always populated with value.

Reply:
not quite sure what u mean .

scrub whole PR for what ?

---
In reply to: [254093004](https://github.com/dotnet/machinelearning/pull/2393#discussion_r254093004) [](ancestors = 254093004)

Reply:
If you add anywhere else that comment to InputColumnName inside ColumnInfo, you need to change it.
If you didn't add it in other places, you just need to fix this one.

---
In reply to: [254094414](https://github.com/dotnet/machinelearning/pull/2393#discussion_r254094414) [](ancestors = 254094414,254093004)

Reply:
So it should just say 

            /// <summary>
            /// Name of column to transform. 
            /// </summary>

?

---
In reply to: [254094755](https://github.com/dotnet/machinelearning/pull/2393#discussion_r254094755) [](ancestors = 254094755,254094414,254093004)

Reply:
@artidoro what do you put in them? Sorry, it fills like billions of same PRs, and it's hard to find for me, but I know what Artidoro address that in his PR recently.

---
In reply to: [254096126](https://github.com/dotnet/machinelearning/pull/2393#discussion_r254096126) [](ancestors = 254096126,254094755,254094414,254093004)

Reply:
Yes that's how I fixed it!